src/client/java/teammates/client/scripts/helper/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,41 @@
+package teammates.client.scripts.helper;
+
+/**
+ * Utility class that provides methods to help with migrating data to its desanitized form.
+ * Contains a private constructor to prevent instantiation of the class.
```

Comment:
I used a private constructor as generally other utility classes in production code do this as well. I initially thought there are no benefits to letting others instantiate this class, but I see now that it can potentially be useful. I will change it to a public constructor.